Constructor ${si.func}, is throwing exception ${si.var} in ${si.filename}, line ${si.line}.